components/content-service/pkg/layer/provider.go

```diff
@@ -227,6 +227,116 @@ func (s *Provider) GetContentLayer(ctx context.Context, owner, workspaceID strin
 	return nil, nil, xerrors.Errorf("no backup or valid initializer present")
 }
 
+// GetContentLayerPVC provides the content layer for a workspace that uses PVC feature
+func (s *Provider) GetContentLayerPVC(ctx context.Context, owner, workspaceID string, initializer *csapi.WorkspaceInitializer) (l []Layer, manifest *csapi.WorkspaceContentManifest, err error) {
+	span, ctx := tracing.FromContext(ctx, "GetContentLayer")
+	defer tracing.FinishSpan(span, &err)
+	tracing.ApplyOWI(span, log.OWI(owner, workspaceID, ""))
+
+	defer func() {
+		// we never return a nil manifest, just maybe an empty one
+		if manifest == nil {
+			manifest = &csapi.WorkspaceContentManifest{
+				Type: csapi.TypeFullWorkspaceContentV1,
+			}
+		}
+	}()
+
+	// check if workspace has an FWB
+	var (
+		bucket = s.Storage.Bucket(owner)
+		mfobj  = fmt.Sprintf(fmtWorkspaceManifest, workspaceID)
+	)
+	span.LogKV("bucket", bucket, "mfobj", mfobj)
+	manifest, _, err = s.downloadContentManifest(ctx, bucket, mfobj)
+	if err != nil && err != storage.ErrNotFound {
+		return nil, nil, err
+	}
+	if manifest != nil {
+		span.LogKV("backup found", "full workspace backup")
+
+		l, err = s.layerFromContentManifestPVC(ctx, manifest, csapi.WorkspaceInitFromBackup, true)
+		return l, manifest, err
+	}
```

Comment:
We probably want to keep the FWB and PVC paths separate for now.
Please remove this.